dlopen the sonamed libs directly, like it has already been done for libGL.so.1,
as we don't want to Depend on the -dev packages for the .so files:
- libosmesa.so.7: Mesa-libOSMesa-7.10-1
- libGLESv2.so.2: Mesa-libGLES-7.11-0.20110227.1
- libEGL.so.1: Mesa-libEGL-7.10-1

--- chromium-browser-14.0.803.0~svn20110625r90472/src/ui/gfx/gl/gl_implementation_linux.cc.orig	2011-06-23 04:01:00.000000000 +0200
+++ chromium-browser-14.0.803.0~svn20110625r90472/src/ui/gfx/gl/gl_implementation_linux.cc	2011-06-25 06:28:03.219139002 +0200
@@ -61,7 +61,7 @@
       }
 
       base::NativeLibrary library = LoadLibrary(
-          module_path.Append("libosmesa.so"));
+          module_path.Append("libosmesa.so.7"));
       if (!library)
         return false;
 
